### PR TITLE
Munitions construction and mapping fixes

### LINKED ISSUE #357: Munitions construction bugs
## Testmerges:
#344 #125 

I'll be fixing this, just keeping track.
- Guns deconstruct into south-facing position
- Munition console linking is still in the wrong direction
- Railgun overlays aren't getting deleted when a gun is deconstructed
- Make it harder to accidentally deconstruct railguns

### PR DESCRIPTION
**Pull request by covertcorvid**
## About The Pull Request
Fixes some issues with munitions construction.
Fixes #357 

## Why It's Good For The Game
Makes things work right, with bonus multitool console linking

## Changelog
:cl:
add: Ship weapons can now be linked to munitions consoles with a multitool
fix: Railguns can no longer be deconstructed without first putting them into maintenance
fix: Railguns and torpedo launchers now deconstruct in the correct direction
fix: Railgun directional subtypes are now the right directions
/:cl:

### CODE REVIEW

File: nsv13/code/modules/munitions/munitions_computer.dm [hunk 1]
```diff
@@ -28,17 +29,32 @@
...
+			SW.linked_computer = src
 
 /obj/machinery/computer/ship/munitions_computer/Destroy()
 	. = ..()
 	if(SW)
 		SW.linked_computer = null
 
+/obj/machinery/computer/ship/munitions_computer/attackby(obj/item/P, mob/user, params)
+	// Using a multitool lets you link stuff
+	if(P.tool_behaviour == TOOL_MULTITOOL)
```
Comment (Jalleo) [inline nsv13/code/modules/munitions/munitions_computer.dm]: multitool_act()

File: nsv13/code/modules/munitions/ship_weapons/_ship_weapon.dm [hunk 2]
```diff
@@ -137,15 +137,30 @@
 	if(!linked)
 		get_ship()
 
+	if(I.tool_behaviour == TOOL_MULTITOOL)
```
Comment (Jalleo) [inline nsv13/code/modules/munitions/ship_weapons/_ship_weapon.dm]: see above

**Comment by Cenrus** [comment]
Railgun/Torp frames that aren't bolted to the floor can be pushed around, causing direction weirdness

**Comment by covertcorvid** [comment]
@Cenrus Not sure what kind of resolution you're hoping for - should they not be un-anchor-able at all, or is it just a matter of making sure they can be rotated?

**Comment by Jalleo** [comment]
What Cenrus likely means is what happens when you use a wrench on a constructed ship weapon after screwdriving its panel open.

**Comment by Cenrus** [comment]
It just felt a bit wrong because the frames are so large, I was wondering if it was intentional.

**Comment by Cenrus** [comment]
Merge in time for the next test, I'm tired of MTs deconstructing the railgun by accident

**Pull request merged**